feat: add changelog action (#1023)
ansys/pyansys-geometry: CHANGELOG.md

# CHANGELOG

> This project uses [*towncrier*](https://towncrier.readthedocs.io/) and the changes for the upcoming release can be found in <https://github.com/ansys/{repo-name}/tree/main/changelog.d/>.
> 

Reviewer: ```suggestion
This project uses [towncrier](https://towncrier.readthedocs.io/) to generate changelogs. You can see the changes for the upcoming release in <https://github.com/ansys/{repo-name}/tree/main/changelog.d/>.

```
We let the link be enough emphasis for a new product, so I removed the italics.
